### PR TITLE
Adding Job management APIs

### PR DESCRIPTION
**Pull request by shahnupur1901**
Includes:

Get Job status, configuration, metrics and overview.
Upload, Plan and run a jar.
Stop gracefully or terminate a job
Get information of all running jobs.
TODO:

Better parameterization of requests can be done.
Adding jars to run.

### CODE REVIEW

File: DjangoProject/business_logic/__pycache__/es_business_logic.cpython-313.pyc
Comment (SilverNarcissus) [inline DjangoProject/business_logic/__pycache__/es_business_logic.cpython-313.pyc:1]: Seems there is no need to upload this bin file.

File: DjangoProject/business_logic/__pycache__/__init__.cpython-313.pyc
Comment (SilverNarcissus) [inline DjangoProject/business_logic/__pycache__/__init__.cpython-313.pyc:1]: Seems there is no need to upload this bin file.

**Review by SilverNarcissus — COMMENTED** [review]
Brilliant job! Only 2 small comments.

File: DjangoProject/business_logic/__pycache__/es_business_logic.cpython-313.pyc
Comment (shahnupur1901) [reply DjangoProject/business_logic/__pycache__/es_business_logic.cpython-313.pyc:1]: Deleted.

File: DjangoProject/business_logic/__pycache__/__init__.cpython-313.pyc
Comment (shahnupur1901) [reply DjangoProject/business_logic/__pycache__/__init__.cpython-313.pyc:1]: Deleted.

**Review by SilverNarcissus — APPROVED** [review]

**Pull request merged**